src/models/user.js

```diff
@@ -44,14 +45,29 @@ export default (sequelize, DataTypes) => {
     hsesUserId: {
       type: DataTypes.STRING,
       unique: true,
+      allowNull: false,
+    },
+    hsesUsername: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      unique: true,
```

Comment:
Assuming hses usernames will be unique, right? (verifying since we already have unique `hsesUserId ` )

Reply:
They definitely should be, but since this is just an informational field for us it would probably make sense to remove our validations on it.

Reply:
Actually, I'm going to leave it since `bootstrapAdmin` queries by the username field I want to make sure we aren't surprised at who is impacted by running that script.